target/arm/cpu.c

```diff
@@ -217,7 +248,14 @@ static void arm_cpu_reset(DeviceState *dev)
         } else {
             env->pstate = PSTATE_MODE_EL1h;
         }
+
+#ifdef TARGET_CHERI
+        reset_capregs(env);
+        env->pc.cap = cc128_make_max_perms_cap(0, cpu->rvbar, CAP_MAX_TOP);
```

Comment:
```suggestion
        env->pc.cap = CAP_cc(make_max_perms_cap)(0, cpu->rvbar, CAP_MAX_TOP);
```
so it doesn't need to be changed once we make cc128 and "cc128m" separate